index.js

```diff
@@ -33,21 +16,7 @@ mongoose.connect(db_url, {
     process.exit();
 });
 
-cron.schedule("* 6 * * *", function() {
-    console.log("running daily at 6");
-    for(var i = 0; i < process.env.TIMES;i++){
-        var d = new Date();
-        const code = d.getTime(); 
-        code = code%100;   
-        const short_url = "http://shortened.com/" + code;
-        console.log(short_url);
-        const note = new newSchema({
-            urlCode: code,
-            shortUrl: short_url
-        });    
-        note.save();
-    }
-});
+require('./cron/cronfile')(app);
 
 app.listen(PORT, () => {
```

Comment:
what if this service goes down on production?
how will you get to know that?
Any alerts?